nitro-protocol/src/index.ts

```diff
@@ -45,3 +47,5 @@ export * from './contract/vouchers';
 export {Uint256, Bytes32} from './contract/types';
 
 export * from './channel-mode';
+
+export {OutcomeShortHand, AssetOutcomeShortHand, computeOutcome} from '../test/test-helpers';
```

Comment:
I think I agree - it's probably cleaner if contents of `/test` are not exported. These are potentially useful helper interfaces though, and could be relocated somewhere in `/src` if they want to be part of the exports.

How / where are you intending to use these as exports?

Reply:
> How / where are you intending to use these as exports?

We want to use them in our ForceMoveApp tests as a quick way to assemble some outcome. For now, a function with similar logic to these helpers was created and is used.

> These are potentially useful helper interfaces though, and could be relocated somewhere in /src if they want to be part of the exports.

I don't think we should put them directly in `/src`, but maybe rather `/src/test`.

Reply:
> I don't think we should put them directly in `/src`, but maybe rather `/src/test` or `/src/helpers`.

:+1: I probably like helpers better, because it's a bit more generic. We have [similar structs](https://github.com/statechannels/go-nitro/blob/9302887765e13861d61906ae4c90a22d8dcc5ac4/channel/consensus_channel/consensus_channel.go#L379) in our go code which provide better dev ergonomics for off-chain manipulation.